dev.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3
+
+import argparse
+from glob import glob
+import shlex
+import subprocess
+from pathlib import Path
+import json
+import sys
+
```

Comment:
Can you add a short docstring explaining the purpose of this script? Is it something we expect users to use, or is it just for Streamlit devs? (Should we include info about it in the repo's README?)

Reply:
Thanks. Added.